src/main/java/de/champonthis/ghs/server/Application.java

```diff
@@ -8,21 +8,19 @@
 import java.nio.file.Files;
 import java.nio.file.StandardCopyOption;
 import java.util.Arrays;
-
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.builder.SpringApplicationBuilder;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.PropertySource;
-import org.springframework.context.annotation.PropertySources;
 import org.springframework.context.support.PropertySourcesPlaceholderConfigurer;
 import org.springframework.core.io.FileSystemResource;
 
 /**
  * The Class Application.
  */
 @SpringBootApplication
-@PropertySources({ @PropertySource("classpath:default.properties"),
```

Comment:
Can you add some comments here to explain the changes? What is the improvement?

Reply:
SonarLint (a static code analyzer) just says it's not necessary since java 8. I did not know this either.

[Link to the explanation](https://rules.sonarsource.com/java/RSPEC-1710).

Quote:
"Before Java 8 if you needed to use multiple instances of the same annotation, they had to be wrapped in a container annotation. With Java 8, that’s no longer necessary, allowing for cleaner, more readable code.
Note that this rule is automatically disabled when the project’s sonar.java.source is lower than 8."

Reply:
Ah cool, very good to know, didn't knew that either.